Tools/auterion/upload_skynode.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+DIR="$(dirname $(readlink -f $0))"
+DEFAULT_AUTOPILOT_HOST=10.41.1.1
+DEFAULT_AUTOPILOT_PORT=33333
+DEFAULT_AUTOPILOT_USER=auterion
+
+for i in "$@"
+do
+    case $i in
+        --file=*)
+        PX4_BINARY_FILE_ARGUMENT=" -f ${i#*=}"
+        ;;
+        --default-ip=*)
+        DEFAULT_AUTOPILOT_HOST="${i#*=}"
+        ;;
+        --default-port=*)
+        DEFAULT_AUTOPILOT_PORT="${i#*=}"
+        ;;
+        --default-user=*)
+        DEFAULT_AUTOPILOT_USER="${i#*=}"
+        ;;
+        --revert)
+        REVERT_AUTOPILOT_ARGUMENT=" -r"
```

Comment:
```suggestion
        REVERT_AUTOPILOT_ARGUMENT=-r
```

Reply:
Added to https://github.com/PX4/PX4-Autopilot/commit/b9c4dc8b97c81af9fdd2bea41554a33f3e3cd561